templates/frontend/pages/aboutEditorialTeamBio.tpl

```diff
@@ -22,7 +22,9 @@
 	</h2>
 
 	<div class="editorialTeamBioContent">
+		<img src="{$baseUrl}/{$publicfiles}/{$editor->_data['profileImage']['uploadName']}?{""|uniqid}" alt="{translate key="user.profile.form.profileImage"}" />
```

Comment:
Accessing private datastructures directly is risky.  Internal data representation may change from version to version, and the internal datastructure may be functionally marked as private at any time.  Instead, you should use the publicly exposed methods and properties for the data object.

See, for example:
https://github.com/pkp/pkp-lib/blob/397b2ec2cbe3bd75e4908344cc1512e39dca9add/classes/user/form/PublicProfileForm.inc.php#L125
and
https://github.com/pkp/pkp-lib/blob/397b2ec2cbe3bd75e4908344cc1512e39dca9add/templates/user/publicProfileForm.tpl#L51